pkg/ovs/openflow/ofctrl_flow.go

```diff
@@ -134,9 +136,16 @@ func (f *ofFlow) CopyToBuilder(priority uint16, copyActions bool) FlowBuilder {
 		matchers: f.matchers,
 		protocol: f.protocol,
 	}
+	if copyActions {
+		newFlow.dropFlag = f.dropFlag
+	}
 	return &ofFlowBuilder{newFlow}
 }
 
+func (f *ofFlow) GetDropFlag() bool {
+	return f.dropFlag
+}
```

Comment:
```
func (f *ofFlow) IsDropFlow() bool {
	return f.isDropFlow
}
```
to be more straightforward?

Reply:
Sounds good. Refactored this to IsDropFlow and f.isDropFlow.